src/LoginServer/World/Character.cs

```diff
@@ -64,5 +65,33 @@ public int[] GetEquipmentProperties()
 			// TODO: This needs to return the actual properties of equipment which have variable lengths.
 			return this.GetEquipIds();
 		}
+
+		/// <summary>
+		/// Sets the default equipment for a character.
+		/// </summary>
+		/// <returns></returns>
+		public bool SetDefaultEquipment()
```

Comment:
The logic here should really only be invoked during character creation, and I thought I might place it in the constructor. However, we also use the constructor when restoring from the data persistence layer (I think ORMs and data mappers use reflection to avoid this problem). Let me know your thoughts.

Reply:
I think there's little need to worry, because where is this gonna be called anyway, except for on creation? If someone went in there and just called it somewhere else for some reason, they might even have a reason for it.

If you want to be 100% sure I would either create a bool that is only not set for new characters, and set at the end of this and for existing characters, to check if it was called already, or check if Equip is empty.